frontend/src/core/models/board/board-model.ts

```diff
@@ -0,0 +1,6 @@
+import { TaskInfoModel } from './task-Info-model';
```

Comment:
task-info-model

Reply:
Do you want to rename the model 'TaskInfoModel' on 'Task-Info-Model'?

Reply:
I'm talking about the file name ('./task-info-model'). Stick to kebab-case, pls